include/odp/api/spec/schedule_types.h

```diff
@@ -173,6 +173,73 @@ extern "C" {
  */
 typedef int odp_schedule_prio_t;
 
+/**
+ * Region specific cache stashing configuration
+ *
+ * Region specific cache stashing configuration for different cache levels.
+ * Application can, for example, configure caching of certain portions of a
+ * region to L2 while configuring another portion to be cached to L3 or
+ * alternatively caching to both levels by configuring overlapping offsets and
+ * byte counts.
+ */
+typedef struct odp_cache_stash_region_t {
+	/** L2 cache stashing */
+	struct {
+		/** Enable or disable cache stashing for level */
+		odp_bool_t enable;
+
+		/** Offset into a region to start caching from */
+		uint32_t offset;
+
+		/** Bytes to cache
+		 *
+		 *  Depending on the implementation, this might be rounded up
+		 *  to a more suitable boundary.
+		 */
+		uint32_t bytes;
```

Comment:
Suggesting "len" instead of "bytes". It's commonly used for (byte) length in the API.